data/hails.txt

```diff
@@ -1187,6 +1275,8 @@ phrase "friendly militia"
 		"while we are still flying"
 		"while we are still breathing"
 		"while we are here"
+		"as long as we are alive"
+		"without going through us first"
```

Comment:
"Not a single innocent life will be destroyed without going through us first!"

So... they're saying you need to obtain their permission before killing innocent life? :wink:

Reply:
You're so critical dude, you make me want to drop the project tbqh, as I'm just trying to be nice and help. Yes, no one will harm the innocents so long as the militia exists. Stop twisting it, it doesn't say "without our permission."

Reply:
I'm being critical because this is a high quality game, and frankly this PR is not of the same high quality. There is a 0% change that Michael is going to merge this in its current state. I'm trying to help you get it merge-worthy, which is why I'm spending half my afternoon going through it again and again to help you polish it up and match the tone, mood, and atmosphere of the game. 

If you aren't willing to do that, then maybe you should drop it.

Reply:
praising one for the fact he made some content is nice and all doesn't help, this criticism is to ensure all additional content matches the quality of the game

Reply:
I appreciate the feedback, but one nit picks. This'll likely be the last thing I help with as I'm just trying to pitch some time for a game I enjoyed. Tbh if you guys want, you could scrap all the changes. But obviously I'd rather you didn't. 

I have no problem with constructive criticism, but I do not appreciate the condescending manner one person is giving it as.

Reply:
don't get us wrong, we appreciate you're taking some time to help develop the game, we're reviewing your changes to make sure the typos are gone, it has correct grammar, it doesn't have clunky hails etc.

Pointedstick didn't mean to sound condescending, he's even spending half his afternoon to help you get your additions added, so please just drop the "negativity" as we're all trying to help you :)

Reply:
I believe you, but the way it was done was offensive to me. I do not hate being told I made a mistake, but there's helping, and nit picking on how a pirate might not speak proper English and they have to use the "proper" words. Not everyone will say you're rather than you are or you were. Or how the militia wouldn't let an innocent get harmed so long as they're alive. It's not a permission thing.

Ty for clarification, but I don't want to come in here and change stuff. If that's how he does things,  then w/e. I just wanted to pay for the game I got for free.

Reply:
just contributing is appreciated
some hails are a bit hard to make though, like these Pirate ones

the reason for "you're" over "you are" is because pirates would speak in an informal way, and English people usually say "you're" unless they're in some formal, say, meeting

Reply:
I appreciate the contribution, I really do. I'm sorry if I came off as rude. But this is just how it works when you propose things in an open-source community. It's customary to make small, conservative changes at first. That way, you can get a sense of how the community operates and what kinds of changes are acceptable, and you also establish your credibility over time. Every project and every community will have a tone, a culture, a set of things that aren't even considered.

You're welcome here as a contributor, but don't try to jam a round peg into a square hole, especially when established contributors are giving you a thumbs-down. A community-oriented review process is necessarily a bracing experience from time to time. :slightly_smiling_face: But don't take it personally! We're just trying to help polish up your contribution so it can make it in, because there's some good stuff in here. But it won't ever make it into the game if it's alongside a bunch of stuff that doesn't cut the mustard.

Reply:
I was not offended by you Kiled :) Only pointed tbqh. Everyone else here even when saying I did something wrong wasn't hyper critical. I may continue at some point, but at this point I'd rather just let Pointed fix all the problems he said I caused in the hails.txt and make it on his own release, as he doesn't really like anything I do. My quarrel is exclusively with how he handles telling me I did something wrong. idk any of you guys, and i'm sure you guys keep in touch. So idk how you guys behave normally. I'm a stranger to this community just making a branch to help pay for the game. While I'm sure everyone is nice, this is just my first impression. All more dramatic than I wanted it to be, since these comments aren't very good at timing messages, and quickly asking questions. 

I'm sorry that I am not experienced as Pointed would like, and I'm sorry that him and I do not think the same. If he has a problem with my writing, he can edit it himself at this point as most things I've added, he has had a problem with. I would much rather be a part of this community, but it's hard to understand what's going on if I don't know how everyone behaves, cause surely you know each other and are very used to one another.

Reply:
>Do you guys use a discord, specifically the dev community for endless sky?

@greatkek https://discord.gg/u6KepgG. Most of the people who have provided feedback are here.

Reply:
Yeah, I'm here. Can't tell who the people are who have helped in the project though without just asking them. tyvm, but I think anyone who works on this project should at least have a chatroom in that server to easily bounce ideas off each other rather than trying to find a comment as these pages get longer and longer. Maybe I'm not using github right, but it seems sorta hard to chat with specific people over specific things. 

Oh well, tyvm for sharing the discord! Hopefully anyone who isn't already there joins it :)